### ISSUE TITLE
Bug: Code exercises fails accessibility tests

### ISSUE THREAD

**Issue by techtolentino** [Qiskit/platypus#1299]
### Steps to reproduce the problem

If one of the code exercises has internal code that causes the block to be scrollable, it will fail the Cypress test.

For example, if the Scratchpad has comment-instructions _within_ the code editor, it'll fail with an a11y violation.

### What is the current behavior?

The Cypress test will fail if a page is tested that includes a scrollable code exercise

### What is the expected behavior?

Code exercise blocks should pass a11y tests

### Screenshots

![Screen Shot 2022-07-07 at 5 53 34 AM](https://user-images.githubusercontent.com/6276074/178035387-da64014d-ad66-4ae2-9f5d-d68c4e406cd1.png)

**Comment by eddybrando** [comment]
This is quite tricky.

CodeMirror adds a tabindex attribute with the value -1, which is the reason why the test complains, since the scrollable part is removed from the default navigation flow.

But in this case it might make sense because the "tab" in the CodeMirror text area is used as in a text editor and not for navigation.

But then again, how is a user navigating with the keyboard supposed to exit the text area once they land there?

Note: I was able to replicate this issue doing the following:

1. In **cypress/integration/accessibility.spec.js**, change the visited page like this: `cy.visit('/course/ch-labs/lab-1-quantum-circuits')`
2. Run `npm run test:debug`
3. Only run the accessibility tests

**Comment by techtolentino** [comment]
> But then again, how is a user navigating with the keyboard supposed to exit the text area once they land there?


Just wanted to note that in @korgan00's PR where he updates [CodeMirror to v6](https://github.com/Qiskit/platypus/pull/1333), a user _can_ get out of the focus trap.

Once a user is in a code cell:
- they can `tab` to indent
- they can `shift` + `tab` to back-indent
- they can press `esc` then `tab` move focus onto the other code cell buttons...

This is an awesome update for accessibility here... 👏 

**Screenshot showing how to escape the focus trap**
https://user-images.githubusercontent.com/6276074/178556663-0fe36a50-abf4-4760-9e43-095cfd45dbb4.mov


[Documentation on CodeMirror tab-handling](https://codemirror.net/examples/tab/)

**Comment by techtolentino** [comment]
Now that we've passed the original failing test, _and_ we have an accessibility improvement with the use of v6, I think this bug is safe to close.
